core/src/main/java/org/apache/any23/rdf/Any23ValueFactoryWrapper.java

```diff
@@ -181,6 +182,22 @@ public Literal createLiteral(String pref, IRI value) {
         return wrappedFactory.createLiteral(pref, value);
     }
 
```

Comment:
I don't know what I'm doing here.  This needs to be reviewed and fixed by someone who does.